server/src-lib/Hasura/RQL/DDL/Schema/Table.hs

```diff
@@ -116,8 +116,8 @@ validateTableConfig
 validateTableConfig tableInfo (TableConfig rootFlds colFlds) = do
     withPathK "custom_root_fields" $ do
       sc <- askSchemaCache
-      let defRemoteGCtx = scDefaultRemoteGCtx sc
-      validateCustomRootFlds defRemoteGCtx rootFlds
+      let allRemoteGCtx = scAllRemoteGCtx sc
```

Comment:
I am not 100% sure if this should be `scAllRemoteGCtx` or simply `adminGCtx`

Reply:
I think it is `GCtx` of all remote schemas, which doesn't have any Postgres related `GCtx`. Hence `scAllRemoteGCtx`. 

Am I right @tirumaraiselvan ?

Reply:
Yes, that's what scAllRemoteGCtx means but I am not sure if it should be that or if it should be GCtx of all (including PG)